frontend/src/app/trees/forests/forest-finder/forest-finder.component.html

```diff
@@ -17,6 +17,15 @@
       <input class="usa-button" type="submit" value="Go to selected forest" />
     </form>
     <hr>
-    <app-map-view [forests]="forests"></app-map-view> 
+    <div *ngFor="let i of rows" class="usa-grid">
+      <div *ngFor="let forest of forests | slice:(i*itemsPerRow):(i+1)*itemsPerRow">
+        <div class="usa-width-one-half forest-image-container">
+          <h5>{{forest.forestName}} National Forest</h5>
```

Comment:
As general feedback, note that not all national forests are named "X National Forest," e.g. Lake Tahoe Basin Management Area, Pawnee National Grassland, and the Midewin National Tallgrass Prairie. Templating like this is likely to lead to problems, unless you already know that every forest that will be included adheres to the "X National Forest" standard.